src/main/api-service.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Printf("Hello World!\n")
+}
```

Comment:
Let's start with at least an enumeration of all the included functions in the API

Reply:
Thanks. We have a newer pull request now. I will update it later today.